util/get-supported-module-env.js

```diff
@@ -4,95 +4,82 @@ const path = require('path');
+    'silverstripe/gha-merge-up',
+    'silverstripe/gha-pull-request',
+    'silverstripe/gha-run-tests',
+    'silverstripe/gha-tag-release',
+    'silverstripe/gha-trigger-ci',
+    'silverstripe/gha-update-js',
+    // tooling
+    'silverstripe/cow',
+    'silverstripe/rhino',
+    'silverstripe/github-issue-search-client',
+    'silverstripe/module-standardiser',
+    'silverstripe/silverstripe-tx-translator',
+    // extra bits and bobs
+    'silverstripe/.github',
+    'silverstripe/eslint-config',
+    'silverstripe/webpack-config',
+    'silverstripe/silverstripe-module',
+    'silverstripe/api.silverstripe.org',
+    'silverstripe/doc.silverstripe.org',
+    'silverstripe/silverstripe-userhelp-content',
+    'silverstripe/demo.silverstripe.org',
+    'silverstripe/silverstripe-frameworktest',
+  ];
+
+  const allModules = {};
+  for (const ref of extraRepos) {
+    allModules[ref] = { github: ref };
+  }
 
-// Modules which are maintained by the product team focused at 'core'.
-// This is an internal workflow aspect which allows this team to use this app
-// for their issue triage. It should match the zenhub.com board,
-// minus repos on github.com/silverstripe-security.
-const coreProductTeamRepos = [
-  'silverstripe/api.silverstripe.org',
-  'silverstripe/cow',
-  'silverstripe/demo.silverstripe.org',
-  'silverstripeltd/open-sourcerers',
-  'silverstripe/recipe-cms',
-  'silverstripe/recipe-core',
-  'silverstripe/recipe-plugin',
-  'silverstripe/silverstripe-admin',
-  'silverstripe/silverstripe-asset-admin',
-  'silverstripe/silverstripe-assets',
-  'silverstripe/silverstripe-behat-extension',
-  'silverstripe/silverstripe-campaign-admin',
-  'silverstripe/silverstripe-cms',
-  'silverstripe/silverstripe-config',
-  'silverstripe/silverstripe-errorpage',
-  'silverstripe/silverstripe-framework',
-  'silverstripe/silverstripe-frameworktest',
-  'silverstripe/silverstripe-graphql',
-  'silverstripe/silverstripe-graphql-devtools',
-  'silverstripe/silverstripe-installer',
-  'silverstripe/silverstripe-postgresql',
-  'silverstripe/silverstripe-reports',
-  'silverstripe/silverstripe-serve',
-  'silverstripe-themes/silverstripe-simple',
-  'silverstripe/silverstripe-siteconfig',
-  'silverstripe/silverstripe-sqlite3',
-  'silverstripe/silverstripe-testsession',
-  'silverstripe/silverstripe-upgrader',
-  'silverstripe/silverstripe-versioned',
-  'silverstripe/silverstripe-versioned-admin',
-  'silverstripe/vendor-plugin',
-  'silverstripe/webpack-config',
-  'silverstripe/developer-docs'
-];
+  // Merge in suported modules from all relevant major release lines.
+  // For modules which appear in more than one release line, prefer the most recent version.
+  const urls = [
+    'https://raw.githubusercontent.com/silverstripe/supported-modules/4/modules.json',
+    'https://raw.githubusercontent.com/silverstripe/supported-modules/5/modules.json',
+  ];
+  for (const url of urls) {
+    await getRelevantModules(url, allModules);
+  }
```

Comment:
```suggestion
  for (let url of urls) {
    await getRelevantModules(url, allModules);
  }
```

Reply:
Why let? Const is correct here.

Reply:
Either works, just `let` just seems more intuitive cos it's going to keep changing as you iterate through the values. `const` just looks wrong to me.

I also just generally hate seeing `const` everywhere, espeicially when it's used in place of `function`

/rant :-)